app/models/tutorial.rb

```diff
@@ -3,4 +3,9 @@ class Tutorial < ApplicationRecord
   acts_as_taggable_on :tags, :tag_list
   accepts_nested_attributes_for :videos
   scope :non_classroom, -> {where classroom: false}
+
+  def has_videos?
+    return true if videos.count > 0
+    false
```

Comment:
I think this can be simplified in one of two ways:
```ruby
def has_videos?
    videos.count > 0
end
```
or:
```ruby
def has_videos?
   videos.any?
end
```

Reply:
Changes have been pushed up. 

```
* Listening on tcp://127.0.0.1:55248
.......................................................................................

Finished in 11.82 seconds (files took 3.58 seconds to load)
89 examples, 0 failures

Coverage report generated for RSpec to /Users/mattsimon/turing/brownfield-of-dreams/coverage. 284 / 340 LOC (83.53%) covered.
```